js/bybit.js

```diff
@@ -5133,38 +5525,48 @@ module.exports = class bybit extends Exchange {
         const toId = this.safeString (accountTypes, toAccount, toAccount);
         const currency = this.currency (code);
         const amountToPrecision = this.currencyToPrecision (code, amount);
-        const request = {
-            'transfer_id': transferId,
-            'from_account_type': fromId,
-            'to_account_type': toId,
-            'coin': currency['id'],
-            'amount': amountToPrecision,
-        };
-        const response = await this.privatePostAssetV1PrivateTransfer (this.extend (request, params));
+        let method = undefined;
+        [ method, params ] = this.handleOptionAndParams (params, 'transfer', 'method', 'privatePostAssetV1PrivateTransfer'); // v1 preferred atm, because it supports funding
```

Comment:
ok in this case we should require `method` inside params, and also allow setting it inside options 

```Javascript
options: {
     transfer: {
               method: 
      }
}

Reply:
may I ask for more clarification? `handleOptionAndParams` does what you say in the backgrounds. Do I miss smth?

Reply:
@ttodua yeah sorry I misread it 😅 , did not notice you were using handleOptionAndparams already